### PR TITLE
Sf 6913 | cli | fix security vulnerabilities, do upgrades

### PR DESCRIPTION
**Pull request by MicheleBuselli**
Unable to fully upgrade copy-webpack-plugin (https://skysync.atlassian.net/browse/SF-7005)
Unable to fully upgrade yargs; upgraded to 15.4.1 but latest version is 16.0.3 (https://skysync.atlassian.net/browse/SF-7004)
Unable to upgrade mocha; test failure on azure (https://skysync.atlassian.net/browse/SF-7003)

Tested using npm run build, run test, and running different skysync-cli commandline commands.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,35 +55,35 @@
...
+    "v8flags": "^3.2.0",
+    "yargs": "^15.4.1"
   },
   "devDependencies": {
     "@types/expect.js": "^0.3.29",
-    "@types/mocha": "^5.2.7",
-    "@types/node": "^12.7.8",
+    "@types/mocha": "^8.0.3",
+    "@types/node": "^14.10.2",
     "copy-webpack-plugin": "^5.1.1",
```
Comment (bentatum) [inline package.json:71]: I was able to fix some vulnerabilities and still make it pass in Azure by doing a minor upgrade to v5.1.2.

https://github.com/portalarchitects/skysync-design-system/pull/570/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R67

**Review by bentatum — CHANGES_REQUESTED** [review]

**Comment by bentatum** [comment]
I'm getting a failure on `npm test`.

```
 110 passing (5s)
  1 failing

  1) formatDate
       relative
         should format days-distant dates:

      Error: expected 'Apr 06, 01:00 PM' to sort of equal 'Apr 06, 1:00 PM'
      + expected - actual

      -Apr 06, 01:00 PM
      +Apr 06, 1:00 PM
```

File: package.json (hunk 1 above)
Comment (MicheleBuselli) [reply package.json:71]: I noticed that - can do the same.

**Comment by MicheleBuselli** [comment]
I get the same failure locally but haven't figured out why yet. However, tests pass in Azure.

**Comment by bentatum** [comment]
Differing node versions, perhaps.

File: package.json (hunk 1 above)
Comment (MicheleBuselli) [reply package.json]: Thanks. Upgraded to 5.1.2 and that got rid of the last high security vulnerability.

**Review by bentatum — APPROVED** [review]

**Pull request merged**